mlx_graphs/utils/transformations.py

```diff
@@ -92,7 +93,7 @@ def to_adjacency_matrix(
 
     Args:
         edge_index (mlx.core.array): a [2, num_edges] array representing the source and target nodes of each edge
```

Comment:
what do you think of providing a consistent doc description for all `edge_index` occurrences? We may do that as well for `node_features` and `edge_features`? We now have many different doc descriptions to describe the same thing.

Reply:
I actually thought about it as well but I'm not sure where to put it at this stage as the documentation is not published. Let me add it to the README and to the first page of the docs, what do you think @TristanBilot ?

Reply:
Ok we can do it later, we just have to remind that to have consistent docs in all methods, and yeah go for the README if you want!

Reply:
I added it to the README and the docs. Let's decide later whether to remove the extensive descriptions from the individual classes/methods docs.